bindata/network/ovn-kubernetes/002-rbac.yaml

```diff
@@ -65,6 +65,14 @@ rules:
 - apiGroups: ['authorization.k8s.io']
   resources: ['subjectaccessreviews']
   verbs: ['create']
+- apiGroups: [certificates.k8s.io]
+  resources: ['certificatesigningrequests']
+  verbs:
+  - create
+  - get
+  - delete
```

Comment:
Probably just needs get & create?

Reply:
Currently, I delete and recreate the certificate every time the Daemonset is started,